### PR TITLE
Bump Dependencies to use `@ValidVesselIMONumber`

### PR DESCRIPTION
**Pull request by nt-as**
Signed-off-by: Niels Thykier <nt@asseco.dk>

### CODE REVIEW

**Review by HenrikHL — DISMISSED** [review]

**Review by HenrikHL — DISMISSED** [review]

File: src/main/java/org/dcsa/ovs/service/impl/TimestampServiceImpl.java [hunk 1]
```diff
@@ -56,6 +72,8 @@
         String modeOfTransport = timestamp.getModeOfTransport() != null ? timestamp.getModeOfTransport().name() : null;
 
         return transportCallRepository.getTransportCall(timestamp.getUNLocationCode(), timestamp.getFacilitySMDGCode(), modeOfTransport, timestamp.getVesselIMONumber())
+                // We should create the transport call if it is missing, but we are missing that feature.
+                .switchIfEmpty(Mono.error(new ResponseStatusException(HttpStatus.NOT_IMPLEMENTED)))
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/ovs/service/impl/TimestampServiceImpl.java:76]: Wouldn't creating TransportCall be part of this PR?

Comment (nt-as) [reply src/main/java/org/dcsa/ovs/service/impl/TimestampServiceImpl.java:76]: No, it requires the new publisher layout as we need carrier SMDG code.

File: src/main/java/org/dcsa/ovs/service/impl/TimestampServiceImpl.java [hunk 2]
```diff
@@ -43,6 +43,22 @@
...
+            return Mono.error(new ResponseStatusException(HttpStatus.NOT_IMPLEMENTED));
+        }
+        if (timestamp.getModeOfTransport() == null) {
+            if (timestamp.getVesselIMONumber() == null) {
+                // OVS 2.0.0 Spec says optional, operations event says mandatory.  The latter wins for now.
+                return Mono.error(new ResponseStatusException(HttpStatus.NOT_IMPLEMENTED));
+            }
+            // Assume vessel IMO number implies VESSEL as mode of transport.
+            timestamp.setModeOfTransport(DCSATransportType.VESSEL);
+        }
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/ovs/service/impl/TimestampServiceImpl.java:61]: Now would be the time if we should update the API spec... I suppose this is only concerning OperationsEvents?

Comment (nt-as) [reply src/main/java/org/dcsa/ovs/service/impl/TimestampServiceImpl.java:61]: I do not disagree but lets add it to "the list".

**Review by HenrikHL — DISMISSED** [review]

**Review by douradofrazer — APPROVED** [review]

**Pull request merged**